lib/core/network/executor.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:chopper/chopper.dart' show Response;
+import 'package:coffeecard/core/errors/failures.dart';
+import 'package:dartz/dartz.dart';
+import 'package:logger/logger.dart';
+
+class Executor {
```

Comment:
you can look at where and how its used, no need to maintain documentation. Otherwise the class could be renamed to NetworkExecutor or RequestExecutor. I personally this its perfectly clear what it does when looking at the implementation